beacon-chain/builder/testing/mock.go

```diff
@@ -37,15 +41,18 @@ func (s *MockBuilderService) SubmitBlindedBlock(_ context.Context, _ interfaces.
 		}
 		return w, s.ErrSubmitBlindedBlock
 	}
-	w, err := blocks.WrappedExecutionPayloadCapella(s.PayloadCapella)
+	w, err := blocks.WrappedExecutionPayloadCapella(s.PayloadCapella, big.NewInt(0))
 	if err != nil {
 		return nil, errors.Wrap(err, "could not wrap capella payload")
 	}
 	return w, s.ErrSubmitBlindedBlock
 }
 
 // GetHeader for mocking.
-func (s *MockBuilderService) GetHeader(context.Context, primitives.Slot, [32]byte, [48]byte) (builder.SignedBid, error) {
+func (s *MockBuilderService) GetHeader(ctx context.Context, slot primitives.Slot, hr [32]byte, pb [48]byte) (builder.SignedBid, error) {
+	if slots.ToEpoch(slot) >= params.BeaconConfig().CapellaForkEpoch {
```

Comment:
Don't we have helpers for this stuff instead of manual conditionals?

Reply:
We do not. We use these manual conditionals across the codebase. We certain can clean up but I prefer to do this in another PR